build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -2279,6 +2334,50 @@ message CacheCapabilities {
   // [ContentAddressableStorage.SpliceBlob][build.bazel.remote.execution.v2.ContentAddressableStorage.SpliceBlob]
   // operation.
   bool blob_splice_support = 10;
+
+  // The chunking configuration of the server.
+  ChunkingConfiguration chunking_configuration = 11;
+}
+
+// The chunking configuration of the server.
+message ChunkingConfiguration {
+  // If any of the advertised parameters are not within the expected range,
+  // the client SHOULD ignore FastCDC chunking function support.
+  message FastCDCParams {
+    // The normalization level for the FastCDC chunking algorithm.
```

Comment:
We should consider having just `avg_chunk_size_bytes`, and all the other parameters are set to defulats and documented.